main.ts

```diff
@@ -263,14 +282,18 @@ export default class ReadwiseMirror extends Plugin {
           last_highlight_at: last_highlight_at ? this.formatDate(last_highlight_at) : '',
           source_url: source_url,
           tags: this.formatTags(tags),

```

Comment:
After further reflections and looking at other plugins that import tags, I came to the conclusion that
- quotes are the wrong way to solve the problem of invalid tags in frontmatter, and
- an option to return the tags without hash would make more sense.